modules/configuring-firewall.adoc

```diff
@@ -5,16 +5,99 @@
 [id="configuring-firewall_{context}"]
 = Configuring your firewall for {product-title}
 
-Before you install {product-title}, you must configure your firewall to access
-Red Hat Insights.
+Before you install {product-title}, you must configure your firewall to grant access to the sites that {product-title} requires.
 
 .Procedure
 
-* Set your firewall to allow the following host names and ports on the outgoing
-network firewall:
+. Whitelist the following registry URLS:
 +
-----
-cert-api.access.redhat.com:443
-api.access.redhat.com:443
-infogw.api.openshift.com:443
-----
+[cols="3,4",options="header"]
+|===
+|URL | Function
+|`registry.redhat.io`
+|Provides core container images
+
+|`quay.io`
+|Provides OLM content
+
+|`sso.redhat.com`
+|The `https://cloud.redhat.com/openshift` site uses authentication from `sso.redhat.com`
+|===
+
+. Whitelist any site that provides resources for a language or framework that your builds require.
+
+. If you do not disable Telemetry, you must grant access to the following host names and ports to access Red Hat Insights:
++
+[cols="3,4",options="header"]
+|===
+|URL | Function
+
+|`cert-api.access.redhat.com:443`
+|Required for telemetry
+
+|`api.access.redhat.com:443`
+|Required for telemetry
+
+|`infogw.api.openshift.com:443`
+|Required for telemetry
+|===
+
+. If you use Amazon Web Services (AWS), Microsoft Azure, or Google Cloud Platform (GCP) to host your cluster, you must grant access URLs that provide the cloud provider API and DNS for that cloud:
++
+[cols="2a,8a,8a",options="header"]
+|===
+|Cloud |URL |Function
+
+.2+|AWS
+
+|`route53.amazonaws.com`
+|Allows the Cluster Ingress Operator to creates wildcard alias records for routes
+
+|`amazonaws.com`
+|Required to access AWS services and resources, including s3, iam, and `quay-registry.s3`
+
+|GCP
+|`www.googleapis.com/dns/v1/projects`
+|cluster-ingress-operator: Creates wildcard alias records for routes.
+
+.3+|Azure
```

Comment:
For Azure it is management.azure.com and we should point here: https://docs.microsoft.com/en-us/rest/api/azure/